src-tauri/src/lib.rs

```diff
@@ -314,6 +319,97 @@ pub fn launch_northstar(
     ))
 }
 
+pub fn get_titanfall_proton() -> Option<String> {
```

Comment:
Maybe add a [doc comment](https://doc.rust-lang.org/reference/comments.html#doc-comments) (`///`) that shortly explains what the function does ^^

Reply:
Still valid btw :P